src/query/management/src/role/role_mgr.rs

```diff
@@ -158,7 +158,24 @@ impl RoleApi for RoleMgr {
             res.ok_or_else(|| ErrorCode::UnknownRole(format!("Role '{}' does not exist.", role)))?;
 
         match seq.match_seq(&seq_value) {
```

Comment:
why not just modify `impl TryFrom<Vec<u8>> for RoleInfo ` ?

Reply:
because this: 

```
Using this TryFrom is not very obvious about what is done when converting OwnershipInfo to bytes.

I would recommend removing it and use serde_json or serialize_struct dierectly.
```

I mark this resolved.